### PR TITLE
frontend/feat: #2061 Access ride management in unserviceable location

### PR DESCRIPTION
**Pull request by dhruvSHA256**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [x] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

### Additional Changes

- [ ] This PR modifies the database schema (database migration added)
- [ ] This PR modifies dhall configs/environment variables
- [ ] This PR contains API breaking changes

## Motivation and Context

https://github.com/nammayatri/nammayatri/issues/2061

## Checklist

- [ ] I formatted the code and addressed linter errors `./dev/format-all-files.sh`
- [ ] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable
- [ ] No leak detected in leakcanary

### CODE REVIEW

File: Frontend/android-native/app/src/user/nammaYatri/res/drawable/ny_ic_map_blur.png
Comment (witcher-shailesh) [inline Frontend/android-native/app/src/user/nammaYatri/res/drawable/ny_ic_map_blur.png:1]: @dhruvSHA256 can you compress the file size and keep it below 30-50kb?

File: Frontend/ui-customer/src/Components/IndividualRideCard/View.purs [hunk 1]
```diff
@@ -365,14 +365,15 @@ viewDetailsAndRepeatRide push state =
       , height WRAP_CONTENT
       , text $ getString VIEW_DETAILS
       , color Color.blue900
-      , padding $ Padding 26 18 50 3
+      , if getValueToLocalStore IS_SERVICEABLE == "true" then padding $ Padding 26 18 50 3 else padding $ Padding 26 18 26 3
```
Comment (witcher-shailesh) [inline Frontend/ui-customer/src/Components/IndividualRideCard/View.purs:368]: Why is SharedPref variable required? Can we use state variables itself for the same?

Comment (dhruvSHA256) [reply Frontend/ui-customer/src/Components/IndividualRideCard/View.purs]: as the individual ride component are show using a  PrestoList, I tried settings value of state variable of a component, based on value of state variable of the screen the component is in, by specifying it in a transformer in Frontend/ui-customer/src/Screens/CustomerUtils/MyRidesScreen/Controller.purs:208
```haskell
myRideListTransformer :: MyRidesScreenState -> Array RideBookingRes -> Array IndividualRideCardState
myRideListTransformer state listRes = filter (\item -> (item.status == "COMPLETED" || item.status == "CANCELLED")) (map (\(RideBookingRes ride) ->
  let
    fares = getFares ride.fareBreakup
    ...
  , isSrcServiceable: state.data.isSrcServiceable
}) (listRes))
```
then access it in component like  (./src/Components/IndividualRideCard/View.purs:393)
```haskell
viewDetailsAndRepeatRideShimmer :: forall w. IndividualRideCardState -> PrestoDOM (Effect Unit) w
viewDetailsAndRepeatRideShimmer state =
  linearLayout
  [ width MATCH_PARENT
  , height WRAP_CONTENT
  , gravity CENTER_HORIZONTAL
  , margin $ MarginTop 15
  ][  sfl $ textView (
      [ width WRAP_CONTENT
      , height WRAP_CONTENT
      , text $ getString VIEW_DETAILS
      , color Color.grey900
      , background Color.grey900
      , padding $ Padding 10 3 10 3
      , PrestoList.visibilityHolder "shimmerVisibility"
      ] <> FontStyle.body1 LanguageStyle)
    , linearLayout 
      [ width $ V 1
      , height if os == "IOS" then (V 10) else MATCH_PARENT
      , margin $ MarginHorizontal 40 40
      , background Color.grey900
      , visibility if state.isSrcServiceable then VISIBLE else GONE
      ][]
    , sfl $ textView (
      [ width WRAP_CONTENT
      , height WRAP_CONTENT
      , text $ getString REPEAT_RIDE
      , PrestoList.visibilityHolder "shimmerVisibility"
      , color Color.grey900
      , background Color.grey900
      , padding $ Padding 10 3 10 3
      , visibility if state.isSrcServiceable then VISIBLE else GONE
      ] <> FontStyle.body1 LanguageStyle)
  ]
```
but the value is not  updating,

Comment (dhruvSHA256) [reply Frontend/ui-customer/src/Components/IndividualRideCard/View.purs]: update: I have fixed this by updating value of dummy card value in my ride screen handler

File: Frontend/android-native/app/src/user/nammaYatri/res/drawable/ny_ic_map_blur.png
Comment (witcher-shailesh) [inline Frontend/android-native/app/src/user/nammaYatri/res/drawable/ny_ic_map_blur.png:1]: @dhruvSHA256 can we reduce the size of these assets?

File: Frontend/ui-customer/src/Components/IndividualRideCard/View.purs [hunk 2]
```diff
@@ -365,14 +365,15 @@ viewDetailsAndRepeatRide push state =
       , height WRAP_CONTENT
       , text $ getString VIEW_DETAILS
       , color Color.blue900
-      , padding $ Padding 26 18 50 3
+      , if state.isSrcServiceable then padding $ Padding 26 18 50 3 else padding $ Padding 26 18 26 3
```
Comment (witcher-shailesh) [inline Frontend/ui-customer/src/Components/IndividualRideCard/View.purs:368]: We already have a variable with which we were handling this case and were showing the full screen popUp, can we use the same one?

File: Frontend/ui-customer/src/Strings/EN.js [hunk 3]
```diff
@@ -215,7 +216,7 @@ const englishStrings = {
   "BOOK_RIDE_": "Book Ride",
   "CANCEL_AUTO_ASSIGNING": "Cancel Auto Assigning",
   "LOCATION_UNSERVICEABLE": "Location unserviceable",
-  "CURRENTLY_WE_ARE_LIVE_IN_": "We are not live in your area yet!",
+  "CURRENTLY_WE_ARE_LIVE_IN_": "We are not live in your area yet!\n\n You can access ride history and other settings from the menu",
```
Comment (witcher-shailesh) [inline Frontend/ui-customer/src/Strings/EN.js:219]: This text is not updated in other files, KA, ML, BA, FR?

**Review by witcher-shailesh — CHANGES_REQUESTED** [review]

File: Frontend/ui-customer/src/Strings/EN.js (hunk 3 above)
Comment (dhruvSHA256) [reply Frontend/ui-customer/src/Strings/EN.js:219]: done

File: Frontend/ui-customer/src/Components/IndividualRideCard/View.purs (hunk 2 above)
Comment (dhruvSHA256) [reply Frontend/ui-customer/src/Components/IndividualRideCard/View.purs:368]: I didn't get it, on home screen we were showing popup using `showlocUnserviceablePopUp`

**Review by witcher-shailesh — APPROVED** [review]

**Review by witcher-shailesh — APPROVED** [review]

**Pull request merged**